instance/tests/test_github.py

```diff
@@ -93,7 +93,7 @@ def test_get_pr_by_number(self):
             '**Description**\r\n\r\nHello!\nDesc with unicode «ταБЬℓσ»\r\n'
             '- - -\r\n**Settings**\r\n```yaml\r\nEDXAPP_FEATURES:\r\n  ALLOW: true\r\n```')
         self.assertEqual(pr.number, 8474)
-        self.assertEqual(pr.fork_name, 'open-craft/edx-platform')
+        self.assertEqual(pr.fork_name, 'edx/edx-platform')
```

Comment:
@antoviaque I believe this change will, in fact, fix the URL, as it will result in a URL like 'https://github.com/edx/edx-platform/pull/4' rather than 'https://github.com/open-craft/edx-platform/pull/4' . The code here uses 'fork-name' to populate the URL.

I suspect what you actually want is for me to rename that attribute, since, after all, it's no longer the fork's name. Does that make sense?